plugins/notifications/ClickAppNotifications.qml

```diff
@@ -26,8 +26,8 @@ ItemPage {
 
     property var entry
     property int entryIndex
-
```

Comment:
keep this line, it's a good practice to separe the public properties from the internal ones